whitelist_python.py

```diff
@@ -0,0 +1,3 @@
+import collections
+
+collections.defaultdict(list).default_factory
```

Comment:
I propose to have both the example code and the whitelist code in one file, i.e., whitelist_python.py should be 

```
import collections

collections.defaultdict(list).default_factory = None
collections.defaultdict(list).default_factory
```

The .txt file can be removed. The test_whitelist_python_function() test should test that vulture finds no dead code in whitelist_python.py.